core/src/main/java/pl/agh/edu/actor/utils/LanguageLabel.java

```diff
@@ -0,0 +1,24 @@
+package pl.agh.edu.actor.utils;
+
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+
+import pl.agh.edu.actor.HotelSkin;
+import pl.agh.edu.language.LanguageChangeListener;
+import pl.agh.edu.language.LanguageManager;
+
+public class LanguageLabel extends Label implements LanguageChangeListener {
```

Comment:
nie wiem czy z tą klasą nie powininiśmy uchwycić wszystkich aspektów zmiana rozdzielczości, czcionki itd itp bo nam się trochę duplikuje z CustomLabel, ale jak dla mnie to następny task na refactor, ale tak bdw